bin/helm-deploy

```diff
@@ -41,6 +37,21 @@ if [ "${HELM_TEMPLATE_ONLY}" != "true" ]; then
   helm init --client-only
 fi
 
+detect_tiller_namespace() {
+    # If there is more than one tiller, then auto-detection will not work.
+    mapfile -t TILLERS < <(kubectl get deploy --all-namespaces -l app=helm,name=tiller | awk 'NR > 1 {print $1}')
+    if [ ${#TILLERS[@]} -ne 1 ]; then
```

Comment:
Should probably be `-gt 1`, since the zero case is handled below

Reply:
👍 Fixed